### PR TITLE
Modify transit overlay SVG icon styling

### PR DESCRIPTION
**Pull request by AdrianaCeric**
**Description:**
Icon SVG styling can be inconsistent for custom icons. These styles help fix that across route viewer, stop viewer and mobile view.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/viewers/RouteRow.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ export const OperatorImg = styled.img`
 `
 
 export const ModeIconElement = styled.span`
-  height: 22px;
+  height: 28px;
```
Comment (binh-dam-ibigroup) [inline lib/components/viewers/RouteRow.js:42]: This is slightly interfering with how the route viewer for configs that use smaller icons (the bus icon can have too much offset wrt to the text, see screnshot):

image

**Review by binh-dam-ibigroup — APPROVED** [review]
I'll let the second reviewer weigh in regarding the icon vertical offset that this PR is creating.

File: lib/components/viewers/RouteRow.js (hunk 1 above)
Comment (AdrianaCeric) [reply lib/components/viewers/RouteRow.js:42]: Great catch! Thanks for testing all cases. Fixed in https://github.com/opentripplanner/otp-react-redux/pull/824/commits/3f0b7eeed81fc7cab19f793e3a9ef9219f16bcff
image

**Review by philip-cline — APPROVED** [review]
LGTM with the vertical fix

**Pull request merged**